aiohttp/websocket.py

```diff
@@ -172,6 +173,7 @@ def WebSocketParser(out, buf):
                     Message(OPCODE_BINARY, data, ''), len(data))
 
 
+native_byteorder = sys.byteorder
```

Comment:
in that operation, byte order is irrelevant, since actual numeric value is not used in that code.